packages/validator/templates/OIS2C&S.json

```diff
@@ -0,0 +1,136 @@
+{
+	"__actions": [
+		{
+			"__insert": {
+				"__target": "config",
+				"__value": {
+					"id": "{FILL}",
```

Comment:
I'm not sure what `{FILL}` is ... is this a placeholder for you? i.e. like a `TODO`?

Reply:
Some values are not present in OAS, and need to be filled manually, Burak suggested to use `{FILL}` for these values.